### PR TITLE
fix: repeat 6185 interactive wrapper height in apps

### PR DESCRIPTION
**Pull request by parkjh**
Screenshot 2019-05-15 at 16 03 43

### CODE REVIEW

**Comment by parkjh** [comment]
need some help testing this on device as it's difficult to duplicate the bug on emulator...

**Comment by aronshamash** [comment]
@parkjh  you add screenshot for for iOS too ?

File: packages/interactive-wrapper/src/interactive-wrapper.js [hunk 1]
```diff
@@ -52,8 +52,11 @@ class InteractiveWrapper extends Component {
       (e && e.nativeEvent && e.nativeEvent.data) ||
       e.nativeEvent.data === "0"
     ) {
+      const { height: prevHeight } = this.state;
       const height = Number(e.nativeEvent.data);
-      this.setState({ height });
+      if (height !== prevHeight && height > prevHeight) {
```
Comment (nathggns) [inline packages/interactive-wrapper/src/interactive-wrapper.js]: This is probably a deal breaker because interactives can change height during use, and this means that throughout the lifecycle of a interactive they can get bigger, but never smaller which is probably not a good enough user experience. Maybe discuss with @chrishutchinson?

Comment (parkjh) [reply packages/interactive-wrapper/src/interactive-wrapper.js]: @nathggns @chrishutchinson did not know that the interactives could do that, I will look into it again...

**Comment by parkjh** [comment]
> @parkjh you add screenshot for for iOS too ?

ios app hot reloading is not working for me... :(
